app/model/entity/AssignmentSolution.php

```diff
@@ -66,9 +67,23 @@ public function getMaxPoints()
     protected $accepted;
 
     /**
-     * @ORM\Column(type="boolean")
+     * @ORM\Column(type="datetime", nullable=true)
+     * Time when a review was started.
+     */
+    protected $reviewStartedAt = null;
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     * Time when a review was closed (and comments made visible to student).
```

Comment:
I cannot find any logic related to `(and comments made visible to student).` where is this handled?

Reply:
I will rephrase that. When the review is closed, a student is notified via email where all comments are sent.
The only thing that is a little weird here is that the "issuesCount" is updated only after the review is closed (so the students won't start panicking too early when the review is still not finished).